liff-app/liff.js

```diff
@@ -1,5 +1,5 @@
 // User service UUID: Change this to your generated service UUID
-const USER_SERVICE_UUID         = '91E4E176-D0B9-464D-9FE4-52EE3E9F1552'; // LED, Button
```

Comment:
would you kindly please change this back to the default starter UUID

Reply:
Alright. I've reverted this change.